cli/popper/utils.py

```diff
@@ -483,7 +483,8 @@ def exec_cmd(cmd, ignoreerror=False):
         fail("Command '{}' failed: {}\n".format(cmd, ex))
 
     output = output.decode('utf-8')
-
+    if verbose:
+        info("{}: \n{}".format(cmd, output))
```

Comment:
while this works, this is going to print everything at once after the command has finished. Also, commands that fail will result in the code not reaching this point (they exit in line 483 above where the `fail` method is invoked).

Instead, what we would like is to stream the output of the command to `stdout`